cmd/metal-api/internal/service/machine-service.go

```diff
@@ -397,6 +397,18 @@ func (r *machineResource) webService() *restful.WebService {
 		Returns(http.StatusOK, "OK", v1.MachineResponse{}).
 		DefaultReturns("Error", httperrors.HTTPErrorResponse{}))
 
+	ws.Route(ws.POST("/{id}/sshpubkeys").
```

Comment:
We could also just utilize the existing machine update endpoint, which already does things like these in order to keep the API surface smaller.

Reply:
done